package/upgrade/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM registry.suse.com/bci/bci-base:15.4
+# source reference: https://github.com/rancher/elemental-cli/commits/v0.3.1
+# ***NOTE*** Deprecated, we need to use the new elemental-cli in the next release.
+FROM registry.suse.com/rancher/elemental-builder-image/5.3:0.3.1 AS temp_elemental
```

Comment:
The deprecated part is already removed in https://github.com/harvester/harvester/pull/5231, so we can remove them.

Reply:
Removed it. Thanks.